pyproject.toml

```diff
@@ -26,18 +26,25 @@ keywords = [
 classifiers = [
   "Development Status :: 4 - Beta",
   "Intended Audience :: Developers",
-  "Natural Language :: English",
+  "Topic :: Software Development",
   "Framework :: Django",
-  "Framework :: Django :: 4.0",
-  "Framework :: Django :: 4.1",
   "Framework :: Django :: 4.2",
   "Framework :: Django :: 5.0",
+  "Framework :: Django :: 5.1",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3 :: Only",
```

Comment:
No need for this, poetry sets it automatically, just when publishing make sure you are on python 3.13

Reply:
What versions will be set by poetry?

I didn't find anything about it in poetry docs. So I'm not sure if we should make it implicit.

Reply:
From min allowed to the one you use as far as I understand
You can unzip file located in dist with `.whl` extension then 
![image](https://github.com/user-attachments/assets/24cd18b9-7e18-476c-9844-2bb203fc9017)
![image](https://github.com/user-attachments/assets/fb7815a3-d86a-4eee-93b9-c32f40d7a5e2)